src/servers/CMakeLists.txt

```diff
@@ -171,6 +217,7 @@ add_executable(
   ${TRTSERVER_HDRS}
   ${HTTP_ENDPOINT_OBJECTS}
   ${GRPC_ENDPOINT_OBJECTS}
+  ${KFSERVING_HTTP_ENDPOINT_OBJECTS}
```

Comment:
After above change can remove here and below since HTTP_ENDPOINT_OBJECTS will have the correct objects for both V1 and V2 cases.

Reply:
Fixed